README.md

```diff
@@ -78,12 +78,11 @@ Note: `int` is used for brevity but can also mean `long`. Don't worry about it.
+| Key                             | Type                                                                                          | Req?     | Notes                                                                                                                                                                                                                                                                                                                                                                |
+|---------------------------------|-----------------------------------------------------------------------------------------------|----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| :group-id                       | string                                                                                        | required |                                                                                                                                                                                                                                                                                                                                                                      |
+| :shape                          | `:value:`, `[:vector <fields>]`,`[:map <fields>]`, or an arity-1 function of `ConsumerRecord` | optional | If unspecified, channel will contain ConsumerRecord objects. [Examples](#data-shapes)                                                                                                                                                                                                                                                                                |
+| :ketu.source/consumer-decorator | `ConsumerDecorator`                                                                           | optional | Decorates the internal poll function. when provided the decorator `poll!` fn will be called with the following params:<br/>consumer-context: {:ketu.source/consumer consumer}<br/>pool-fn: fn [] -> Iterable<ConsumerRecord> <br/>Returns an iterable collection of consumerRecord.<br/>The decorator should call the poll-fn on behalf of the consumer source.<br/> |
```

Comment:
This part of the table seems complex, especially with the `<br/>` usage.  
Alternative suggestion: Leave the "`ConsumerDecorator`", link to the relevant namespace in the source code, and enhance the docstring there.